pkg/ddevapp/global_dotddev_assets/commands/host/phpmyadmin

```diff
@@ -0,0 +1,39 @@
+#!/bin/bash
+
+## #ddev-generated: If you want to edit and own this file, remove this line.
+## Description: Prompt downloading phpMyAdmin if it is not installed.
+## Usage: phpmyadmin
+## Example: "ddev phpmyadmin"
+
+if [ "${DDEV_NONINTERACTIVE:-}" != "" ]; then
+  echo "Nothing has been changed."
+  exit 0
+fi
+
+read -p "phpMyAdmin is not installed would you like to install? (Y/n): [Y]" answer
+if [ -z "$answer" ]; then
+    answer="Y"
+fi
+
+case "$answer" in
+    [Yy] | [Yy][Ee][Ss])
+        ddev get ddev/ddev-phpmyadmin
+        echo "phpMyAdmin has been installed and requires a restart."
+        read -p "Would you like me to restart now? (Y/n): [Y]" answerrestart
```

Comment:
In case the user doesn't have to manually enter  ddev restart, then i would suggest to slightly rephrase randys suggestion to: 

```suggestion
        read -p "Would you like to restart DDEV now? (Y/n): [Y]" answerrestart
```